pandas/core/arrays/categorical.py

```diff
@@ -1461,28 +1461,6 @@ def value_counts(self, dropna=True):
 
         return Series(count, index=CategoricalIndex(ix), dtype="int64")
 
-    def get_values(self):
-        """
-        Return the values.
-
-        .. deprecated:: 0.25.0
-
-        For internal compatibility with pandas formatting.
-
-        Returns
-        -------
-        numpy.array
-            A numpy array of the same dtype as categorical.categories.dtype or
-            Index if datetime / periods.
-        """
-        warn(
-            "The 'get_values' method is deprecated and will be removed in a "
-            "future version",
-            FutureWarning,
-            stacklevel=2,
-        )
-        return self._internal_get_values()
-
     def _internal_get_values(self):
```

Comment:
Can you add part of the above docstring to this method? (the interesting parts for internal use, eg what it returns)

(and potentially the same comment for some of the other cases below)

Reply:
good idea